appinventor/appengine/src/com/google/appinventor/client/widgets/LabeledTextBox.java

```diff
@@ -176,7 +176,7 @@ public boolean validate() {
   private void setErrorStyles(boolean validationResult) {
     if (validationResult) {
       textbox.getElement().getStyle().setBorderColor(defaultTextBoxColor);
-      errorLabel.setText("");
+      errorLabel.setText(errorMessage);
```

Comment:
I don't agree with this change. If there wasn't any error (i.e., `validationResult == true`), then there should not be any error messages. This changes the semantics in a way that doesn't make sense to me. Instead, we might want to extend the Validator interface to also have a getWarningMessages method that we can use instead and is more explicit in its naming.

Reply:
@ewpatton 
So correspondingly should I add new warningLabel?
or Continue with errorLabel with warning messages

Reply:
@Ramanathi I would just reuse the error label, but add a new case for changing the outline color to dark yellow to differentiate a warning from an error. If there is an error, that should be shown. Warnings should only be shown if there aren't any errors.